pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -50,9 +51,13 @@ func MakeSegmentStoreStatefulSet(pravegaCluster *api.PravegaCluster) *appsv1.Sta
 			Selector: &metav1.LabelSelector{
 				MatchLabels: util.LabelsForSegmentStore(pravegaCluster),
 			},
-			VolumeClaimTemplates: makeCacheVolumeClaimTemplate(pravegaCluster.Spec.Pravega),
 		},
 	}
+	if util.IsVersionBelow07(pravegaCluster.Spec.Version) {
+		log.Printf("creating volume claim ")
```

Comment:
Either print a more meaningful message or remove this log

Reply:
removed